test/common/utxobased/engine/engine.fixtures/bitcoinTestnet/dummyAddressData.ts

```diff
@@ -0,0 +1,258 @@
+export default {
```

Comment:
All dummy data needs to be refactored to the new currency plugin disk format and then the tests for this data needs to be unskipped.